captum/attr/_core/deep_lift.py

```diff
@@ -55,16 +62,57 @@ def is_output_cloned(output_fn, input_grad_fn):
 
 
 class DeepLift(GradientAttribution):
-    def __init__(self, model):
+    def __init__(self, model: Module) -> None:
         r"""
         Args:
 
             model (nn.Module):  The reference to PyTorch model instance.
         """
         GradientAttribution.__init__(self, model)
         self.model = model
-        self.forward_handles = []
-        self.backward_handles = []
+        self.forward_handles: List[RemovableHandle] = []
+        self.backward_handles: List[RemovableHandle] = []
+
+    @typing.overload
+    def attribute(
+        self,
+        inputs: TensorOrTupleOfTensors,
+        baselines: Optional[
+            Union[int, Tuple[int, ...], Tensor, List[Tuple[int, ...]]]
```

Comment:
This baselines hint should match the ones below, `Union[Tensor, int, float, Tuple[Union[Tensor, int, float], ...]]`